config/packages/ezplatform.yaml

```diff
@@ -56,11 +56,14 @@ ezplatform:
         list: [site]
         groups:
             site_group: [site]
+            site_factory_1st_purpose_group: []
+            site_factory_2nd_purpose_group: []
         default_siteaccess: site
         match:
             URIElement: 1
+            HostElement: 1
```

Comment:
The EzPlatformSiteFactory\SiteAccessMatcher only matchs host, so, lets match subdomain by default.

- admin.localhost will match `admin` siteaccess
- site.localhost will match `site` siteaccess
- localhost, if not used as a public access in Site Factory, will fallback on default siteaccess which is `site`